addons/website_mail/models/mail_message.py

```diff
@@ -42,30 +23,40 @@ def default_get(self, cr, uid, fields_list, context=None):
 
         return defaults
 
-    def _search(self, cr, uid, args, offset=0, limit=None, order=None,
-                context=None, count=False, access_rights_uid=None):
+    description = fields.Char(compute="_compute_description", help='Message description: either the subject, or the beginning of the body')
```

Comment:
Yes sir, in website_mail_channel (https://github.com/odoo/odoo/blob/master/addons/website_mail_channel/views/website_mail_channel.xml#L156) ;)